pkg/unikontainers/hypervisors/vmm.go

```diff
@@ -23,6 +23,16 @@ import (
 	"github.com/urunc-dev/urunc/pkg/unikontainers/unikernels"
 )
 
+// we declare HypervisorConfig struct here to avoid import cycles
+
+// HypervisorConfig struct is used to hold hypervisor specific configuration
+// that is parsed from the urunc config file or state.json annotations
+type HypervisorConfig struct {
+	DefaultMemoryMB int    `toml:"default_memory_mb"`
```

Comment:
I think we should make this uint, instead of int, since we can not have a negative memory value.